internal/jobworker/cgroup/cgroup_test.go

```diff
@@ -0,0 +1,233 @@
+package cgroup
+
+import (
+	"bufio"
+	"errors"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"testing"
+)
+
+func TestCleanup(t *testing.T) {
+	if !isRoot() {
+		t.Skip("must be root to run")
+	}
+
+	dir := t.TempDir()
+	service, err := NewService(WithMountPath(dir))
+	if err != nil {
+		t.Errorf("%+v", err)
+		return
+	}
+
+	if _, err := os.Stat(service.path); err != nil {
+		t.Error(err)
```

Comment:
Use t.Fatal? Could we proceed after this error?

Same for a few others throughout.

Reply:
I've swapped all uses of t.Error to be t.Fatal since tempdirs are being throughout.